src/client/theme-default/composables/outline.ts

```diff
@@ -194,14 +195,19 @@ export function useActiveAnchor(
     }
 
     const activeLink = prevActiveLink
-
     if (activeLink) {
+      const rect = activeLink.getBoundingClientRect()
+      const isInViewPort = rect.top >= 65 && rect.bottom <= innerHeight
       activeLink.classList.add('active')
       marker.value.style.top = activeLink.offsetTop + 39 + 'px'
       marker.value.style.opacity = '1'
+      if (!isInViewPort) {
+        activeLink.scrollIntoView({ block: 'center' })
+      }
     } else {
       marker.value.style.top = '33px'
       marker.value.style.opacity = '0'
+      aside.value.scrollTop = 0
```

Comment:
Hello, may I ask what this line of code is doing? I saw in the developer tools that the aside itself does not generate a scroll bar, so scrollTop remains 0 all the time. Can this piece of code and related parts be removed?